scripts/upgradeable/README.md

```diff
@@ -0,0 +1,21 @@
+The upgradeable variant of OpenZeppelin Contracts is automatically generated from the original Solidity code. We call this process "transpilation" and it is implemented by our [Upgradeability Transpiler](https://github.com/OpenZeppelin/openzeppelin-transpiler/).
+
+When the `master` branch or `release-v*` branches are updated, the code is transpiled and pushed to [OpenZeppelin/openzeppelin-contracts-upgradeable](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable) by the `upgradeable.yml` workflow.
+
+## `transpile.sh`
+
+Applies patches and invokes the transpiler with the necessary command line flags.
```

Comment:
Feel free to adapt this suggestion as you wish, I just feel it's not explicit enough to just say `the necessary contracts`
```suggestion
Applies patches and invokes the transpiler with necessary command line flags to include or exclude certain purpose-specific contracts
```

Reply:
This is fine, I also don't think we need to be exhaustive but the original writing was not saying much 😅 thanks!